src/proof/alf/alf_node_converter.cpp

```diff
@@ -690,5 +692,44 @@ size_t AlfNodeConverter::getOrAssignIndexForConst(Node v)
   return id;
 }
 
+bool AlfNodeConverter::isHandledSkolemId(SkolemFunId id)
+{
+  switch (id)
+  {
+    case SkolemFunId::PURIFY:
+    case SkolemFunId::ARRAY_DEQ_DIFF:
+    case SkolemFunId::DIV_BY_ZERO:
+    case SkolemFunId::INT_DIV_BY_ZERO:
+    case SkolemFunId::MOD_BY_ZERO:
+    case SkolemFunId::SQRT:
+    case SkolemFunId::TRANSCENDENTAL_PURIFY_ARG:
+    case SkolemFunId::QUANTIFIERS_SKOLEMIZE:
+    case SkolemFunId::STRINGS_NUM_OCCUR:
+    case SkolemFunId::STRINGS_NUM_OCCUR_RE:
+    case SkolemFunId::STRINGS_OCCUR_INDEX:
+    case SkolemFunId::STRINGS_OCCUR_INDEX_RE:
+    case SkolemFunId::STRINGS_OCCUR_LEN:
+    case SkolemFunId::STRINGS_OCCUR_LEN_RE:
+    case SkolemFunId::STRINGS_DEQ_DIFF:
+    case SkolemFunId::STRINGS_REPLACE_ALL_RESULT:
+    case SkolemFunId::STRINGS_ITOS_RESULT:
+    case SkolemFunId::STRINGS_STOI_RESULT:
+    case SkolemFunId::STRINGS_STOI_NON_DIGIT:
+    case SkolemFunId::RE_FIRST_MATCH_PRE:
+    case SkolemFunId::RE_FIRST_MATCH:
+    case SkolemFunId::RE_FIRST_MATCH_POST:
+    case SkolemFunId::RE_UNFOLD_POS_COMPONENT:
+    case SkolemFunId::BAGS_DEQ_DIFF:
+    case SkolemFunId::BAGS_MAP_PREIMAGE:
+    case SkolemFunId::BAGS_MAP_PREIMAGE_INJECTIVE:
+    case SkolemFunId::BAGS_MAP_PREIMAGE_SIZE:
+    case SkolemFunId::BAGS_MAP_SUM:
+    case SkolemFunId::TABLES_GROUP_PART:
+    case SkolemFunId::TABLES_GROUP_PART_ELEMENT: return true;
+    default: break;
```

Comment:
Did you miss `SETS_MAP_DOWN_ELEMENT`? 
By the way, there is a typo in its documentation.

Reply:
https://github.com/cvc5/cvc5/blob/main/src/expr/skolem_manager.h#L450
the second argument has index 3 instead of 2

Reply:
Ill fix this on the skolemApi PR.